tests/main/validate-container-happy/test-snapd-validate-container-happy/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: test-snapd-validate-container-happy
+version: 1.0
+apps:
+  validate-container:
+    command: bin/validate-container
+plugs:
+  hell:
```

Comment:
If the intention is to test whether these hellish symlinks are sharable/work via the content interface, then just adding these plugs here is not enough. We would need to add another snap with the hellish symlinks in them, then expose the directory with all those symlinks in them via a content interface slot and then create a second snap with a compatible content interface plug and then connect the two and ensure that the snap with the plug has the expected view

However I'm not sure the intention of testing the content interface as it pertains to these symlinks, since all the content interface is doing is bind mounting one directory over another directory and AFAIK bind mounts don't really care about how hellish symlinks are inside the directory.

If you want to exercise more things with these hellish symlinks, probably the place to look is in the layout code, where we are laying out another symlink on top of one of these hellish symlink, or perhaps laying out a hellish symlink on top of a normal symlink, but still that should work without issues since again bind mounts don't really care about the target of symlinks, just that the things being bind mounted on is of the same type as the source. 

So long story short, I would recommend against testing the content interface stuff in this PR and instead would recommend just sticking to the basics of testing `snap pack` and `snap try` with these symlinks since we do indeed have code in both of those which historically was fragile with respect to things like circular symlinks.

Reply:
Makes sense to add packing the snap with the "hellish" symlinks. Switched the test flow to:

1. Pack the snap with tarred symlinks (as they cannot live in the repo)
2. "try" the snap, untar the symlinks
3. Pack the snap with the untarred symlinks
4. Install the snap and check that symlinks are still pointing to the right location

Also reverted the "twisted" test back to being disabled in the system to decouple from this PR.

Reply:
thanks, can you remove the `plugs` section from this snap.yaml too?